devices/CharacterLcd/custom/SpanChar.cs

```diff
@@ -0,0 +1,175 @@
+﻿namespace System
```

Comment:
missing header

Reply:
I have added a header (and a comment that the autogenerator (where the class was copied from should also handle this)